frontends/article/components/ArticleActions/article-action-icons.tsx

```diff
@@ -87,9 +87,7 @@ export const ArticleActionIcons: FC<IArticleActionsIconsProps> = ({
       <div className={styles.icon} onClick={(): void => likeDislike(1)}>
         {likeIcon}
       </div>
-      {Boolean(likeDislikeCount) && (
-        <span className={styles.reactionsText}>{likeDislikeCount}</span>
-      )}
+      {likeDislikeCount > 0 ? <span className={styles.reactionsText}>{likeDislikeCount}</span> : ''}
```

Comment:
You did NOT understand the issue
now we are calculating the result: likeDislikeCount = likeCount - dislikeCount
we should show only likeCount without subtracting dislikeCount